lib/salus/package_utils/gradle_dependency_parser.rb

```diff
@@ -1,16 +1,11 @@
 module Gradle
-  GRADLE7 = "/opt/gradle/gradle-7.3.3/bin/gradle".freeze
-  GRADLE6 = "/opt/gradle/gradle-6.9.2/bin/gradle".freeze
-
   def gradle_dependencies
     dependency_metadata_regex = /-\s(?<group_id>.+):(?<artifact_id>.+):(?<version>.+)/
 
     # 'gradle dependencies' command needs to be run in the folder where buid.gradle is present.
-    shell_result = run_shell("#{GRADLE7} dependencies")
-
-    shell_result = run_shell("#{GRADLE6} dependencies") if !shell_result.success?
+    shell_result = run_shell("/home/repo/gradlew dependencies")
```

Comment:
Please see my other note about `/home/repo`.  Then check if `@repository.path_to_repo` is accessible from here.